Test: multiple_stderr_warnings_test
Status: SUCCESS
Command: ./stderr_warnings_test.py
Return code: 0
Start time: 2025-04-08 04:42:19.344143
End time: 2025-04-08 04:42:20.389287

--- STDOUT ---
Starting test with multiple stderr warnings...
Continuing normal execution...
More normal output...
Test completed successfully!


--- STDERR ---
First warning message to stderr
Second warning message to stderr
Third warning message to stderr
Fourth warning message to stderr
Fifth warning message to stderr
